source/features/softwrap-code.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+Soft-wrap code inside codeblocks
+*/
+import select from 'select-dom';
+import features from '../libs/features';
+
+const tabSize = 4;
```

Comment:
We leave out spaces to be spaces, as is. But tabs are used to compute how many spaces to match. I don't get what you mean by _detect indentation_, because RGH already sets size of tabs to be 4 using CSS.

Reply:
> We leave out spaces to be spaces, as is

Spaces and tabs might be mixed

> RGH already sets size of tabs to be 4 using CSS.

People customize things https://github.com/lukechilds/github-custom-tab-size

Reply:
Maybe you can select the indentation with Range and [get its width](https://stackoverflow.com/a/28457044/288906)

Reply:
Might've found a better way, please review 2523e31db0104f6988a00eea5269b94389c1cc5b